content/chainguard/administration/terraform-provider/index.md

```diff
@@ -13,6 +13,7 @@ menu:
     parent: "administration"
 toc: true
 weight: 008
+contentType: "tutorial"
```

Comment:
I'm unclear on when something is a product doc vs a tutorial (or something else). I think this is the first `tutorial` I've seen but I think it could go either way here.